todolist/todo_list.go

```diff
@@ -15,57 +15,93 @@ func (t *TodoList) Add(todo *Todo) {
 	t.Data = append(t.Data, todo)
 }
 
-func (t *TodoList) Delete(id int) {
-	i := -1
-	for index, todo := range t.Data {
-		if todo.Id == id {
-			i = index
+func (t *TodoList) Delete(ids ...int) {
+	for _, id := range ids {
+		todo := t.FindById(id)
+		if todo == nil {
+			break
```

Comment:
looks like you're doing `continue` for the other functions.  should this be `continue` as well?